src/components/TopicSection/TopicSection.test.tsx

```diff
@@ -42,11 +53,13 @@ describe("Test topic buttons", () => {
     const topicButton = await screen.findByTestId(MARKETING);
 
     expect(topicButton).toBeTruthy();
+    await filterButtonsToBeEnabled();
```

Comment:
Consider combining filterButtonsToBeEnabled() and topicButton.click() in single function. This way it is required that every test knows it should wait for buttons to become enabled again. 

You can introduce method clickTopicButton("MARKETING") that will wait for it to become available and then issue a click event